gef-ui/server/utils/get-cover-start-date-or-start-of-today.ts

```diff
@@ -1,7 +1,7 @@
 import { parseISO, startOfDay } from 'date-fns';
-import { AnyObject } from '@ukef/dtfs2-common';
+import { Facility } from '../types/facility';
 
-export const getCoverStartDateOrStartOfToday = (facility: AnyObject): Date => {
+export const getCoverStartDateOrStartOfToday = (facility: Facility): Date => {
```

Comment:
Looks like this file doesn't have a unit test 😞 

Can we create a tech debt ticket to add a test?

Reply:
Have added since it was added by my last PR